src/aiq/observability/register.py

```diff
@@ -23,138 +22,26 @@
-            ex,
-            exc_info=True,
-        )
-
-
-class LangfuseTelemetryExporter(TelemetryExporterBaseConfig, name="langfuse"):
-    """A telemetry exporter to transmit traces to externally hosted langfuse service."""
-
-    endpoint: str = Field(description="The langfuse OTEL endpoint (/api/public/otel/v1/traces)")
-    public_key: str = Field(description="The Langfuse public key", default="")
-    secret_key: str = Field(description="The Langfuse secret key", default="")
-
-
-@register_telemetry_exporter(config_type=LangfuseTelemetryExporter)
-async def langfuse_telemetry_exporter(config: LangfuseTelemetryExporter, builder: Builder):
-    """Create a Langfuse telemetry exporter."""
-
-    import base64
-
-    trace_exporter = telemetry_optional_import("opentelemetry.exporter.otlp.proto.http.trace_exporter")
-
-    secret_key = config.secret_key or os.environ.get("LANGFUSE_SECRET_KEY")
-    public_key = config.public_key or os.environ.get("LANGFUSE_PUBLIC_KEY")
-    if not secret_key or not public_key:
-        raise ValueError("secret and public keys are required for langfuse")
-
-    credentials = f"{public_key}:{secret_key}".encode("utf-8")
-    auth_header = base64.b64encode(credentials).decode("utf-8")
-    headers = {"Authorization": f"Basic {auth_header}"}
-
-    yield trace_exporter.OTLPSpanExporter(endpoint=config.endpoint, headers=headers)
-
-
-class LangsmithTelemetryExporter(TelemetryExporterBaseConfig, name="langsmith"):
-    """A telemetry exporter to transmit traces to externally hosted langsmith service."""
-
-    endpoint: str = Field(
-        description="The langsmith OTEL endpoint",
-        default="https://api.smith.langchain.com/otel/v1/traces",
-    )
-    api_key: str = Field(description="The Langsmith API key", default="")
-    project: str = Field(description="The project name to group the telemetry traces.")
-
-
-@register_telemetry_exporter(config_type=LangsmithTelemetryExporter)
-async def langsmith_telemetry_exporter(config: LangsmithTelemetryExporter, builder: Builder):
-    """Create a Langsmith telemetry exporter."""
-
-    trace_exporter = telemetry_optional_import("opentelemetry.exporter.otlp.proto.http.trace_exporter")
-
-    api_key = config.api_key or os.environ.get("LANGSMITH_API_KEY")
-    if not api_key:
-        raise ValueError("API key is required for langsmith")
-
-    headers = {"x-api-key": api_key, "Langsmith-Project": config.project}
-    yield trace_exporter.OTLPSpanExporter(endpoint=config.endpoint, headers=headers)
-
+class FileTelemetryExporter(TelemetryExporterBaseConfig, name="file"):
+    """A logger to write runtime logs to the console."""
 
-class OtelCollectorTelemetryExporter(TelemetryExporterBaseConfig, name="otelcollector"):
-    """A telemetry exporter to transmit traces to externally hosted otel collector service."""
+    filepath: str = Field(description="The file path to log traces.")
```

Comment:
What about overwriting? Should we have a flag to overwrite or possibly this should be a directory for rolling logs?